### PR TITLE
Book Recommendation System using SparkSQL and MLlib- Spark Part 3

### PR DESCRIPTION
**Pull request by keerthivarumbudy**

### CODE REVIEW

**Comment by hectorkambow** [comment]
Plagiarism check done

**Comment by hectorkambow** [comment]
Wanted to double check that the article images do not need any sources - also one of your link does not lead to the correct URL - please add correct link
@keerthivarumbudy

File: articles/sparksql-mllibspark-part-3/index.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+*Note*: In the recent release of Spark 3, the developers have deprecated RDD programming in their Machine Learning libraries.
+
+[Dataframes and Datasets are part of Spark SQL](https://spark.apache.org/docs/latest/sql-programming-guide.html), which is a Spark module for structured data processing. A **Dataset** is a distributed collection of data. Dataset is an interface that adds the benefits such as increased efficiency provided by SparkSQL’s computation engine to RDDs usage of powerful lambda functions and strongly typed data. A **DataFrame** is a *Dataset* organized into named columns.
+Let us see what a dataframe looks like by converting our books dataset (not to be confused with Spark dataset) into a dataframe, and begin coding a book recommender. *Make sure* [*Books*](http://www2.informatik.uni-freiburg.de/~cziegler/BX/BX-CSV-Dump.zip) *dataset is downloaded and placed in the same folder where you will store your PySpark script.*
```
Comment (hectorkambow) [inline articles/sparksql-mllibspark-part-3/index.md:28]: books link is broken - please add correct link

**Comment by hectorkambow** [comment]
👋  Good afternoon (Happy Friday) @nategrieb  - Wanted to ask if you could give this SparkSQL and MLlib article a tech review before we publish whenever you have a moment. 
Thank you 🚀

**Comment by keerthivarumbudy** [comment]
> Wanted to double check that the article images do not need any sources - also one of your link does not lead to the correct URL - please add correct link
> @keerthivarumbudy

@hectorkambow All the images are screenshots from my program, so they were not picked up from any other source. I updated the link.

**Review by nategrieb — APPROVED** [review]
Looks great, good to publish!

**Comment by hectorkambow** [comment]
> Looks great, good to publish!

Thank you v much 🚀 @nategrieb

**Pull request merged**